src/Commands/CasMockServerCommands.php

```diff
@@ -158,29 +158,60 @@ public function list($options = ['format' => 'table']): RowsOfFields {
    * @return int
    *   The status, to be returned as an error code for machine reading:
    *   - 0: The user has been created.
-   *   - 1: An error occurred while creating the user.
+   *   - 1: An attribute option without the attribute name has been passed.
+   *   - 2: An error occurred while creating the user.
    *
    * @command cas-mock-server:user-create
    * @option email The email address for the new mock user
    * @option password The password for the new mock user
+   * @option attribute Additional attributes to be added to the CAS account.
+   *   Multiple attributes can be passed by adding more than one --attribute
+   *   option. Each attribute consists from the attribute name concatenated with
+   *   the attribute value and using the semicolon (:) as separator. Empty
+   *   attribute values are allowed. Unlike the attribute name, the value can
+   *   contain the semicolon (:) char.
    * @aliases casms-uc
    * @usage drush casms-uc myuser --email="user@example.com" --password="mypass"
    *   Creates a new mock user with the user name mockuser, the email address
-   *   mockuser@example.com, and the password mypass
+   *   user@example.com, and the password mypass
+   * @usage drush cas-mock-server:user-create myuser --email="user@example.com" --password="mypass" --attribute=firstname:Joe --attribute=lastname:Doe
+   *   Creates a new mock user with the user name mockuser, the email address
+   *   user@example.com, the password mypass, the 'firstname' attribute Joe, and
+   *   the 'lastname' attribute Doe
    */
-  public function create($username, array $options = ['email' => self::REQ, 'password' => self::REQ]): int {
+  public function create($username, array $options = [
+    'email' => self::REQ,
+    'password' => self::REQ,
+    'attribute' => [],
+  ]): int {
     $user_data = [
       'username' => $username,
       'email' => $options['email'],
       'password' => $options['password'],
     ];
+
+    // Add additional attributes.
+    foreach ($options['attribute'] as $attribute) {
+      // The attribute value could contain the semicolon (:) char.
+      $parts = explode(':', $attribute, 2);
```

Comment:
We could use `list()` here to map the attribute name and value directly, but this is too small a nitpick to block this PR.

Reply:
@pfrenssen 

```
php -r 'list($a, $b) = explode("abc");'
PHP Warning:  explode() expects at least 2 parameters, 1 given in Command line code on line 1
```